tests/test_coordinates_arithmetic.py

```diff
@@ -76,10 +76,12 @@ def test_arithmetic_coordinates_mul_div(other, operator, order):
         new = part1 * part2
         desired = pf.Coordinates([0, 5], [0, 5], [0, 5])
     elif operator == "/":
-        new = part1 / part2
         if order:
+            new = part1 / part2
             desired = pf.Coordinates([0, .2], [0, .2], [0, .2])
         else:
+            with pytest.warns(RuntimeWarning, match='divide by zero'):
```

Comment:
I think we should not check Warnings and errors from other packages, It would be better in this case to just prevent this warning by choosing Coordinates which are not zero, because the idea of the test is, to check the correct behaviour for valid operations, and dividing by 0 is not valid.

Reply:
changed the coordinates to avoid the warning in the first place as suggested